### DISCUSSION TITLE
Could a submit to a repo not affect other sub–dir(s) of the same repo?

### DISCUSSION THREAD

**Discussion by caibinwav123456** [s0/git-publish-subdir-action#66] [Q&A / Help & Support] [answered]
When I submit something to repository A, this action deletes all contents within the desired branch of A before acting push operation (I can see it clearly in the log). I wish to use one repository to store multiple published building artifacts or multiple versions of releases, so is there a way to do this?

**Comment by s0** [comment, marked as answer]
@caibinwav123456 yes. For this you would use the `CLEAR_GLOBS_FILE` as described here: https://github.com/s0/git-publish-subdir-action#custom-clear-operations

**Comment by caibinwav123456** [comment]
I've got it, then if I issue a new release, is there an environment variable that reflects the version or title or any other attribute I write while creating the release, which can be used to identify it (eg. I can put it into a file name)?

Just a naive question~

**Comment by s0** [comment]
Hi @caibinwav123456, not sure exactly what you mean here. Do you mean you're triggering this workflow on releases in GitHub, and you want to get the release information and add that to the commit messages that are sent to the remote repo? That should be possible, if you share your workflow file I may be able to help more?
